packages/website/components/meta-context.js

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+const ContextMeta = React.createContext({});
+
+export default ContextMeta;
```

Comment:
How do you feel about exporting explicitly named provider consumer pairs, i.e. MetaProvider, MetaConsumer

Reply:
```js
const { Provider: MetaProvider, Consumer: MetaConsumer = React.createContext({});

 export MetaProvider;
 export MetaConsumer;
```

Like that?

I'm not super concerned about which way we do this. I have a slight preference for how I wrote it. I will give in to literally any pressure on this.

Reply:
Yeah, like that. I like it simply because it looks more readable when it's used (MetaConsumer vs MetaContext.Consumer)